Localization/Translations/Options/Tracker.lua

```diff
@@ -335,8 +335,8 @@ local trackerOptionsLocales = {
         ["koKR"] = false,
         ["esMX"] = "Minimizar en calabozos",
         ["enUS"] = true,
-        ["zhCN"] = "地下城时最小化",
-        ["zhTW"] = false,
+        ["zhCN"] = false,
```

Comment:
Was the Chinese translation wrong or why was it removed?

Reply:
Can you explain this @class2u?

Reply:
The translation is right. I think I copied and pasted it wrong, my mistake. Sorry.

Reply:
No worries 👍🏻